Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -164,6 +164,13 @@ var ListView = React.createClass({
      */
     renderSectionHeader: PropTypes.func,
     /**
+     * (props) => renderable
+     *
+     * A function that returns the scroll view in which the list rows are
+     * rendered. Defaults to returning a ScrollView with the given props.
+     */
+    renderScrollView: React.PropTypes.func.isRequired,
```

Comment:
Will update with a renamed version.

Regarding `isRequired`, I understand it to mean that the type of `this.props.renderScrollView` is `Function` instead of `?Function` in Flow-speak. So because we're saying the prop is non-null (which is true because of the default value from getDefaultProps), Flow won't complain about `this.props.renderScrollView()` needing a null check.